rust/main/utils/run-locally/src/main.rs

```diff
@@ -448,19 +448,19 @@ fn main() -> ExitCode {
 
```

Comment:
should we add some invariants to the e2e test to make sure that these sealevel events / txs are being indexed? similar to what we do for the EVM things